src/components/Section/VotingStatusSummary.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import Flickity from 'react-flickity-component'
+import Bold from 'decentraland-gatsby/dist/components/Text/Bold'
+import Time from 'decentraland-gatsby/dist/utils/date/Time'
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import useCountdown from 'decentraland-gatsby/dist/hooks/useCountdown'
+import useAbbreviatedNumber from '../../hooks/useAbbreviatedNumber'
+import { ProposalAttributes } from '../../entities/Proposal/types'
+import { ChoiceProgressProps } from '../Status/ChoiceProgress'
+import 'flickity/css/flickity.css'
+import './VotingStatusSummary.css'
+
+const flickityOptions = {
+  initialIndex: 0,
+  cellSelector: '.VotingStatusSummary__CarouselCell',
+  cellAlign: 'center',
+  contain: true,
+  accessibility: true,
+  pageDots: true,
+  wrapAround: false,
+  autoPlay: 10000,
+  groupCells: false,
+  prevNextButtons: false,
+  setGallerySize: true,
+}
+
+export type VotingStatusSummaryProps = Omit<React.HTMLAttributes<HTMLDivElement>, 'children'> & {
+  proposal?: ProposalAttributes | null
+  votes: ChoiceProgressProps[]
+}
+
+export default function VotingStatusSummary({ proposal, votes }: VotingStatusSummaryProps) {
+  const l = useFormatMessage()
+  const vpInFavor = votes[0].power || 0
+  const threshold = proposal?.required_to_pass || 0
+  const neededForAcceptance = threshold - vpInFavor
+  const thresholdReached = threshold <= vpInFavor
+  const endDate = Time.from(proposal?.finish_at)
+  const timeout = useCountdown(endDate)
+  const abbreviatedThreshold = useAbbreviatedNumber(threshold)
+
+  return (
+    <>
+      {timeout.time > 0 && !thresholdReached && (
+        <Flickity className={'VotingStatusSummary__Carousel'} options={flickityOptions}>
+          <div className="VotingStatusSummary__CarouselCell VotingStatusSummary__Container">
+            <div className="VotingStatusSummary__Subtitle">{l('page.proposal_detail.required_vp')}</div>
+            <div className="VotingStatusSummary__Title">{l('general.number', { value: threshold })} VP</div>
+          </div>
+          <div className="VotingStatusSummary__CarouselCell VotingStatusSummary__Container">
+            <div className="VotingStatusSummary__Subtitle">{l('page.proposal_detail.needed_for_acceptance')}</div>
+            <div className="VotingStatusSummary__Title">{l('general.number', { value: neededForAcceptance })} VP</div>
+          </div>
+        </Flickity>
+      )}
+      {timeout.time > 0 && thresholdReached && (
+        <div className="VotingStatusSummary__Container">
+          <div className="VotingStatusSummary__Subtitle">
+            {thresholdReached && l('page.proposal_detail.threshold_reached', { threshold: abbreviatedThreshold })}
+          </div>
+          <div className="VotingStatusSummary__Title">
+            <Bold>{l('page.proposal_detail.time_left_label', { countdown: endDate.fromNow() })}</Bold>
```

Comment:
I think you have to add `true` as an argument in fromNow to avoid the prefix.
<img width="174" alt="image" src="https://user-images.githubusercontent.com/8242162/161071926-bd323644-a13e-43a0-88f6-4f5c68cfcf8b.png">